go.mod

```diff
@@ -1,21 +1,23 @@
 module github.com/nspcc-dev/neofs-sdk-go
 
-go 1.22
+go 1.23.0
```

Comment:
No `.0`, please.

Reply:
added by `go get -u ./...`

Reply:
I know, but it's wrong. And `toolchain` is wrong too. We want 1.23 for a library and we can't dictate any specific toolchains, whatever (compatible) version builds it should be used as is.

Reply:
ok, will drop manually, although this is extremely inconvenient

Reply:
Only happens with incorrect dependencies in fact. I really don't like the whole `toolchain` concept as well as patch versions in `go` specification. `go` is about the language version and it does not change with patch versions. `toolchain` is designed to make stable builds irrespective of local versions, but we specifically have workflows using both supported versions of Go and we want them to produce whatever result they're producing.